event_routing_backends/processors/xapi/transformer.py

```diff
@@ -51,12 +50,15 @@ def base_transform(self):
         """
         Transform the fields that are common for all events.
         """
+        actor = self.get_actor()
+        event_timestamp = self.get_timestamp()
         self.transformed_event = {
-            'actor': self.get_actor(),
+            'actor': actor,
             'context': self.get_context(),
-            'timestamp': self.get_timestamp(),
-            'id': uuid.uuid4()
+            'timestamp': event_timestamp,
         }
+        uuid_str = f'{actor.to_json()}-{event_timestamp}'
```

Comment:
Sorry, one last thing. Since changing anything in these 2 lines or changing the "base_uuid" can invalidate billions of rows in the database can we put in big warning comments in both places to let people know they should not touch them without having a community discussion first?

Reply:
yes makes sense. added warning comment.